sdk/security_keyvault/src/account/operations/list_secrets.rs

```diff
@@ -14,6 +14,7 @@ impl ListSecretsBuilder {
 
             let mut uri = self.client.client.vault_url.clone();
             uri.set_path("secrets");
+            uri.set_query(Some(API_VERSION_PARAM));
```

Comment:
Is this the only operation where this is necessary?

Reply:
It is not.  This should be part of the PR I have open for keyvault.